.github/workflows/ci.yml

```diff
@@ -91,6 +91,7 @@ jobs:
     - run: exec sh -c "nix-instantiate -E 'builtins.currentTime' --eval"
     - run: exec zsh -c "nix-instantiate -E 'builtins.currentTime' --eval"
     - run: exec fish -c "nix-instantiate -E 'builtins.currentTime' --eval"
+    - run: exec bash -c "nix-channel --update && nix-env -iA nixpkgs.hello && hello"
```

Comment:
It's currently broken (because there's no channel added by the installer), but yes. The thing is that if we want to test that the installer works fine, this is an important code path to test, and I'm not sure how we can make it more pure. Is there a way to pin a channel to a specific version?

Reply:
> Is there a way to pin a channel to a specific version?

So a 5mins investigation shows that `nix-channel --add https://releases.nixos.org/nixos/unstable/nixos-23.05pre466020.60c1d71f2ba nixpkgs` seems to do the trick. Is that a stable url that we can rely on? And is that an acceptable way of pinning the channels version?

Reply:
So we we still want this since the "mock channel" test?